domains/transfer/Transfer.API/API/Transfer/TransferAgreementCleanup/Options/TransferAgreementCleanupOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+using System;
+
+namespace API.Transfer.TransferAgreementCleanup.Options;
+
+public class TransferAgreementCleanupOptions
+{
+    public const string Prefix = "TransferAgreementCleanup";
+
+    [Range(typeof(TimeSpan), "00:00:02", "02:00:00")]
```

Comment:
I thought it was every hour, and not every 2 hours?

Reply:
We can configure it to every 2 hours if we want but this will be configured to every hour.